deploy/crds/redhatcop.redhat.io_keepalivedgroups_crd.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
this needs to stay v1beta1 as v1 is only supported in >4.5 .
when you do `operator-sdk generate crds` add the option `crd-version`